.github/scripts/run-tests

```diff
@@ -58,7 +58,9 @@ do
 
     # this is a bit messy and brittle, but certain tests
     # need to be run with specific options
-    if [[ "${TEST}" == *"kernels"* ]]; then
+    if [[ "${TEST}" == *"kernels/test_pos_encoding"* ]]; then
```

Comment:
it looks like we need to restrict "tests/lora/test_layers.py" as well to run on just one device, i.e. `CUDA_VISIBLE_DEVICES=0 pytest tests-outside/lora/test_layers.py` ...